private_repo/ansible/inventory/dev/Lern/common.yml

```diff
@@ -0,0 +1 @@
+../KnowledgePlatform/common.yml
```

Comment:
This wont work right in 5.1.0? There are additional variables. Please check the dev and staging env once and see how its configured.

Reply:
Are you referring to the CSP variables ? If so, it will be reflected after this PR is merged https://github.com/project-sunbird/sunbird-devops/pull/3689/files

Reply:
Ok in that case, is this section added in the templates?
```
## Lern DP
```
Check staging inventory for more details.

Reply:
@keshavprasadms Updated the PR with Lern BB vars, please check